app/code/community/Bolt/Boltpay/Helper/Data.php

```diff
@@ -336,7 +336,7 @@ public function getPaymentBoltpayConfig($configPath, $checkoutType)
         /** @var string $configValue */
         $configValue = Mage::getStoreConfig('payment/boltpay/'.$configPath);
 
-        return ($checkoutType === Bolt_Boltpay_Block_Checkout_Boltpay::CHECKOUT_TYPE_ADMIN) ? '' : $configValue;
+        return ($checkoutType === Bolt_Boltpay_Block_Checkout_Boltpay::CHECKOUT_TYPE_ADMIN) && !Mage::getStoreConfig('payment/boltpay/use_javascript_in_admin') ? '' : $configValue;
```

Comment:
why this is needed?

Reply:
The original developer that implement this refactor missed a long-standing admin option in error.  The option is intended to be self-describing and is explained in the admin interface as follows:

![image](https://user-images.githubusercontent.com/973856/50770322-2c063500-1287-11e9-881c-600a278726d4.png)